src/flask/globals.py

```diff
@@ -49,3 +49,5 @@
 session: SessionMixin = LocalProxy(  # type: ignore[assignment]
     _cv_request, "session", unbound_message=_no_req_msg
 )
+
+_false_attributes: t.List[str] = ["0", "false", "no", "null", "none"]
```

Comment:
`null` and `none` do not make sense here, it's not something people would pass for a boolean option.

Also, this does not belong in this file. `helpers.py` is already he most suitable path.

But considering that it's already in two very simple utility functions, I'm not sure if it even makes much sense to extract this into a separate list (which, by the way, should be a set and not a list)

Reply:
it makes better practice. its not hard coded and better to maintaine.
in addition, i aggree that none and null shouldnt be passed as boolean but you can use None as False in python (in a condition. " if None" is will evaluate to False.